commands/jobs.go

```diff
@@ -78,6 +78,18 @@ func (command *JobsCommand) Execute([]string) error {
 		}
 		row = append(row, statusColumn)
 
+		var currentColumn ui.TableCell
+		if p.NextBuild != nil {
+			currentColumn.Contents = p.NextBuild.Status
+			switch p.NextBuild.Status {
+			case "pending:":
+				currentColumn.Color = ui.PendingColor
+			case "started":
+				currentColumn.Color = ui.StartedColor
+			}
+			row = append(row, currentColumn)
+		}
```

Comment:
I think this should append "n/a" if there's no next build, so there's a consistent column count. This is important for tools like `awk`.

Reply:
changed in commit 87228af